recipes/ddtrace-python.rb

```diff
@@ -6,6 +6,11 @@
 # NB: This recipe is compatible with Chef <= 12 only. If you're using Chef 13 or higher
 # please refer to the README of the cookbook.
 
+log 'message' do
```

Comment:
any reason to use the resource instead of `Chef::Log`? (genuine question, didn't know about the resource)

Reply:
Not any, I've changed to use Chef::Log as I see there is already many usage of it in the repo, I prefer to stick with known things.